### PR TITLE
Added Dhanush Kiran Profile

### PR DESCRIPTION
**Pull request by Dhanu084**

### CODE REVIEW

File: batches/2019/career-camp-21-october-2019/index.html [hunk 1]
```diff
@@ -132,7 +134,24 @@
...
       </div>
     </div>
-    
+    </div>
+    <div class="row">
+    <div class="col">
+
+      <div class="card ml-3 mt-2" style="width: 10rem;">
+        <a href="https://my-resume-68b58.web.app/" target="_blank" class="no-text-decoration">
+          <img src="./students/Dhanush-Kiran/dhanu.jpg" class="card-img-top" alt="profile-pic" style="height: 8.5rem;">
```
Comment (dakshkhetan) [inline batches/2019/career-camp-21-october-2019/index.html]: Remove the style attribute in line number 143.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

File: batches/2019/career-camp-21-october-2019/index.html (hunk 1 above)
Comment (Dhanu084) [reply batches/2019/career-camp-21-october-2019/index.html]: Removed the style attribute

Comment (shivamyth) [reply batches/2019/career-camp-21-october-2019/index.html]: Take a fresh pull and then send a PR. Actually we have changed something in the card check that

**Review by shivamyth — CHANGES_REQUESTED** [review]
Change as requested

**Comment by shivamyth** [comment]
You haven't made any changes to this PR for a very long. Closing it for now.

**Pull request closed without merging**